test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -274,32 +275,6 @@
         (is (= catalogs-response [{:producer_timestamp current-time}]))
         (is (= factsets-response [{:producer_timestamp current-time}]))))))
 
-(deftest migration-in-different-schema
```

Comment:
It took me a while to find it, but this test actually breaks the ability to call `digest` in the database - I can't tell if the issue is because it removes the `pg_trgm` extension, or if it changes the schema on the connection pool to `pdbtestschema`, or both? Either way - my understanding is that we don't support running puppetdb in schema other than `public`, so this test is unnecessary. If we wish to support that in the future, we need a test like this, but there are other issues that will come up.